### ISSUE TITLE
Update it to support the latest Firefox 56 and other Firefox 56 forks

### ISSUE THREAD

**Issue by AM4Ryzen** [3marcusw/simplewhitecompact#10]
Simple White X supports all the way up to 58.0, so it should be easy to take the latest release from [here](https://github.com/johngraciliano/simplewhite/releases/) and make it compact

I'd also like a dark version with the colors from the Dark Moon theme from [here](https://github.com/Lootyhoof/darkmoon), but it's optional, and I wouldn't mind too much, as I've somewhat gotten used to a white browser theme

**Comment by 3marcusw** [comment]
Hello, asdf2345. Unfortunately this project is no longer maintained - feel free to fork it

**Comment by AM4Ryzen** [comment]
> Hello, asdf2345. Unfortunately this project is no longer maintained - feel free to fork it

I'm asking, because I don't know what all I need to modify to do that

**Comment by 3marcusw** [comment]
I think i put the compact customization at https://github.com/3marcusw/simplewhitecompact/blob/a7d76fbaa28bd030bba5c6a6cfb7f6e55c5429a2/whitefox/browser/browser-common.css

**Comment by AM4Ryzen** [comment]
> I think i put the compact customization at https://github.com/3marcusw/simplewhitecompact/blob/a7d76fbaa28bd030bba5c6a6cfb7f6e55c5429a2/whitefox/browser/browser-common.css

I was looking through that, trying to find what all was changed, only to find it was just a bunch of customizations that had its own comments, and it was easy to transplant into the newest Simple White X. I did some customizations like bringing back always showing back and forward buttons too.

Maybe I'll look through Dark Moon, or a different dark fork of Simple White, and see if it works the same, and customize it.

Maybe I'll even release my version as a fork of Simple White X
